script_library/clean-airship.sh

```diff
@@ -98,6 +98,16 @@ if [[ ${clean_action} == *"clean_images"* ]]; then
     fi
 fi
 
+#If jobs are not deleted.
+if [[ ${clean_action} == *"clean_ucp"* ]]; then
+    kubectl get jobs -n ucp | awk '{print$1}'| xargs kubectl delete job -n ucp
```

Comment:
Any reason we need separate if clause for this. Can this be moved before line #91